src/pages/AddPatient.jsx

```diff
@@ -5,237 +5,303 @@ import {
   CardContent,
   Modal,
   TextField,
-  Typography,
+  Typography
 } from "@mui/material";
 import Grid from "@mui/material/Grid2";
-function AddPatient({ open, onClose }) {
+import { addPatient } from "../services/patients";
+import { useMutation, useQueryClient } from "@tanstack/react-query";
+import { useForm } from "react-hook-form";
+import { useState } from "react";
+import Toast from "../components/Toast";
+const AddPatient = ({ open, onClose }) => {
+  const { register, handleSubmit, reset } = useForm();
+  const [openToast, setOpenToast] = useState(false);
+  const [toastMessage, setToastMessage] = useState("");
+  const [toastSeverity, setToastSeverity] = useState("");
+  const queryClient = useQueryClient();
+  function handleReset() {
```

Comment:
I'm not sure this function is being called from anywhere and if so, you could probably just call the reset function directly